nuxeo-drive-core/src/test/java/org/nuxeo/drive/service/TestNuxeoDriveManager.java

```diff
@@ -293,13 +282,14 @@ public void testSynchronizationRootDeletion() throws Exception {
         checkRootsCount(user2, 1);
 
         // check deletion by lifecycle
-        session.followTransition(doc("/default-domain/workspaces/workspace-2/folder-2-1").getRef(), "delete");
+        trashService.trashDocument(folder_2_1);
+        txFeature.nextTransaction();
         session.save();
```

Comment:
You probably can remove the `session.save()` here as you added `txFeature.nextTransaction()`